requirements.txt

```diff
@@ -1,15 +1,14 @@
 # python development requirements for the Datadog Agent
 reno==3.5.0
-docker==6.0.1; python_version >= '3.7'
+docker==6.1.3; python_version >= '3.7'
 docker==5.0.3; python_version < '3.7'
-docker-squash==1.0.9
-dulwich==0.20.45
+docker-squash==1.1.0
+dulwich==0.21.6
 toml==0.10.2
 packaging==21.3
-# more recent boto3 has a bug which generates an import exception.
-# pin until this can be resolved.
-boto3==1.22.4
--r https://raw.githubusercontent.com/DataDog/datadog-agent-buildimages/main/requirements/common.txt
 # urllib3 major version 2, released on May 4th 2023, breaks botocore used
 # by awscli (removed DEFAULT_CIPHERS list from urllib3.util.ssl_)
+# Needs to be declared first otherwise urllib2 is fetched by another module
 urllib3==1.26.15
+-r requirements/common.txt
```

Comment:
@chouetz Removing remote references prevent remote usage, see failures in test-infra-definitions